src/main.cpp

```diff
@@ -10,7 +10,11 @@ All rights reserved (see LICENSE).
 #include <fstream>
 #include <iostream>
 #include <sstream>
-#include <unistd.h>
+#ifndef _WIN32
+  #include <unistd.h>
+#else
+  #include "utils/getopt_win.h"
```

Comment:
urgh.. I did change the include here without utils.. was sure I committed that! will rectify

Reply:
done